### ISSUE TITLE
Webdev broke on getSupportedProtocols?

### ISSUE THREAD

**Issue by mnordine** [dart-lang/webdev#1036]
Steps to reproduce:

- Install Dart 2.8.4
- `pub global activate webdev`

Result:
```
Resolving dependencies...
+ args 1.6.0
+ async 2.4.1
+ browser_launcher 0.1.7
+ build_daemon 2.1.4
+ built_collection 4.3.2
+ built_value 7.1.0
+ charcode 1.1.3
+ checked_yaml 1.0.2
+ collection 1.14.12
+ convert 2.1.1
+ crypto 2.1.5
+ devtools 0.2.5 (0.2.6-dev.1 available)
+ devtools_server 0.2.5 (0.2.6-dev.1 available)
+ devtools_shared 0.2.5 (0.2.6-dev.1 available)
+ dwds 3.1.3
+ file 5.1.0
+ fixnum 0.10.11
+ http 0.12.1
+ http_multi_server 2.2.0
+ http_parser 3.1.4
+ intl 0.16.1
+ io 0.3.4
+ json_annotation 3.0.1
+ logging 0.11.4
+ matcher 0.12.6
+ meta 1.1.8
+ mime 0.9.6+3
+ package_config 1.9.3
+ path 1.7.0
+ pedantic 1.9.0
+ pool 1.4.0
+ pub_semver 1.4.4
+ pubspec_parse 0.1.5
+ quiver 2.1.3
+ shelf 0.7.5
+ shelf_packages_handler 2.0.0
+ shelf_proxy 0.1.0+7
+ shelf_static 0.2.8
+ shelf_web_socket 0.2.3
+ source_maps 0.10.9
+ source_span 1.7.0
+ sse 3.5.0
+ stack_trace 1.9.3
+ stream_channel 2.0.0
+ stream_transform 1.2.0
+ string_scanner 1.0.5
+ term_glyph 1.1.0
+ typed_data 1.1.6
+ usage 3.4.1
+ uuid 2.0.4
+ vm_service 4.1.0
+ watcher 0.9.7+15
+ web_socket_channel 1.1.0
+ webdev 2.5.8
+ webkit_inspection_protocol 0.7.3
+ yaml 2.2.1
Precompiling executables...
Failed to precompile webdev:webdev:
../../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/dwds-3.1.3/lib/src/services/chrome_proxy_service.dart:40:7: Error: The non-abstract class 'ChromeProxyService' is missing implementations for these members:
 - VmServiceInterface.getSupportedProtocols
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class ChromeProxyService implements VmServiceInterface {
      ^^^^^^^^^^^^^^^^^^
../../../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/vm_service-4.1.0/lib/src/vm_service.dart:745:24: Context: 'VmServiceInterface.getSupportedProtocols' is defined here.
  Future<ProtocolList> getSupportedProtocols();
```

**Comment by mnordine** [comment]
`pub global activate webdev 2.5.6` works

**Comment by Nahyako** [comment]
Dude I had the same problem and suffered ours until I found your post. 
Thank you so much!!!!

**Comment by TTFTCUTS** [comment]
If anyone has any further insight on why this is happening, I'd like to know. 

I'm guessing in my case it's a mismatch between the intellij plugin and webdev or something, but it's a pain when it was working correctly just last night.

**Comment by gmpassos** [comment]
Same error here!

[Dart VM version: 2.8.4 (stable) (Wed Jun 3 12:26:04 2020 +0200) on "macos_x64"]

If you restart your computer and your IDE calls
`pub global activate webdev` (like IntelliJ), your are Dummed!

I recommend to add `pub upgrade` to your CI.


```
Failed to precompile webdev:webdev:
/Users/gmp/.pub-cache/hosted/pub.dartlang.org/dwds-3.1.3/lib/src/services/chrome_proxy_service.dart:40:7: Error: The non-abstract class 'ChromeProxyService' is missing implementations for these members:
 - VmServiceInterface.getSupportedProtocols
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class ChromeProxyService implements VmServiceInterface {
      ^^^^^^^^^^^^^^^^^^
/Users/gmp/.pub-cache/hosted/pub.dartlang.org/vm_service-4.1.0/lib/src/vm_service.dart:745:24: Context: 'VmServiceInterface.getSupportedProtocols' is defined here.
  Future<ProtocolList> getSupportedProtocols();
                       ^^^^^^^^^^^^^^^^^^^^^
```

**Comment by gmpassos** [comment]
This issue was resolved with ‘webdev’ 2.5.9

**Comment by mnordine** [comment]
Fix confirmed
